static/docs/user-guide/cache-file-linking.md

```diff
@@ -0,0 +1,93 @@
+# File Link Types for the DVC Cache
+
+File links are entries in the file system that don't necessarily hold the file
+contents, but which point to where the file is actually stored. DVC uses file
+links in your project's workspace to avoid copying them from/to the DVC cache.
```

Comment:
them - sounds like "to avoid copying links"

Reply:
Changed to "avoid duplicating contents that will exist in the DVC cache."